packages/config/src/Config.types.ts

```diff
@@ -65,7 +65,7 @@ export type PublicExpoConfig = Omit<ExpoConfig, 'hooks' | 'ios' | 'android'> & {
   android?: Omit<ExpoConfig['android'], 'config'>;
 };
 
-export type ExpoAppManifest = ExpoConfig & {
```

Comment:
👌

Reply:
later on in this type:

```
ios?: { publishSourceMapPath?: string } & ExpoConfig['ios'];
```

should we update that too?

```
ios?: { publishSourceMapPath?: string } & PublicExpoConfig['ios'];
```

same for android